jobs/schedulers.go

```diff
@@ -103,8 +102,8 @@ func (schedulers *Schedulers) Start() *Schedulers {
 							if scheduler != nil {
 								if scheduler.Enabled(cfg) {
 									if _, err := schedulers.scheduleJob(cfg, scheduler); err != nil {
-										mlog.Warn(fmt.Sprintf("Failed to schedule job with scheduler: %v", scheduler.Name()))
-										mlog.Error(fmt.Sprint(err))
+										mlog.Warn("Failed to schedule job with", mlog.String("scheduler", scheduler.Name()))
+										mlog.Error("error", mlog.Err(err))
```

Comment:
The string "error" seems redundant here. We could change this line to just `mlog.Error(err.Error())`

Reply:
This is exactly what we are trying to avoid now, having error messages without the information well structured, I think the approach proposed by @niklabh is the correct one here, probably, I would change "error" with something more descriptive, something like "Error scheduling a job", I'm not 100% sure why we have a `mlog.Warn` and right after a `mlog.Error`.

Maybe the correct approach is convert both calls to on `mlog.Error` call with all the data.

Something like:

```go
mlog.Error("Failed to schedule job", mlog.String("scheduler", scheduler.Name()), mlog.Err(err))
```

Reply:
done